services/payments/src/simcore_service_payments/api/rest/_acknowledgements.py

```diff
@@ -19,34 +29,82 @@
 
 
 async def on_payment_completed(
-    payment_id: PaymentID, ack: AckPayment, session: SessionData
+    transaction: PaymentsTransactionsDB, rut_api: ResourceUsageTrackerApi
 ):
+    assert transaction.completed_at is not None  # nosec
+    assert transaction.initiated_at < transaction.completed_at  # nosec
+
+    _logger.debug("TODO next PR Notify front-end of payment -> sio ")
```

Comment:
minor: if you think about it this in the end is a comment, that will only be seen in development mode. If you want it to have any meaning it should be raised to INFO or even WARNING.

Reply:
I only want to see it in debug. More details about my debug("TODO") below